org.omg.sysml/.launch/Generate MOF2KerMLText.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication">
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
+        <listEntry value="/org.omg.sysml/xtend-gen/org/omg/sysml/generation/MOF2KerMLText.java"/>
+    </listAttribute>
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
+        <listEntry value="1"/>
+    </listAttribute>
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_ATTR_USE_ARGFILE" value="false"/>
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_SHOW_CODEDETAILS_IN_EXCEPTION_MESSAGES" value="true"/>
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_CLASSPATH_ONLY_JAR" value="false"/>
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
+    <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="org.omg.sysml.generation.MOF2KerMLText"/>
+    <stringAttribute key="org.eclipse.jdt.launching.MODULE_NAME" value="org.omg.sysml"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="SysML.uml&#10;&quot;${workspace_loc:/SysML-v2-Pilot-Implementation/sysml.library/Kernel Libraries/Kernel Semantic Library/KerML.kerml}&quot;"/>
```

Comment:
So this is the slightly transformed MOF model that is not the same as the one in the "model" folder, right?

Reply:
Yes, that is correct.